### PR TITLE
Language selector UI

### LINKED ISSUE #801: Language Selection
**Is your feature request related to a problem? Please describe.**
This is a child issue of #798

**Describe the solution you'd like**
1. Develop language selection page & create change language widgets on certain specific pages to allow users to easily switch language within the app.
2. We prefer to have language selection upon registration. Future changes would occur in the user settings screen

### PR DESCRIPTION
**Pull request by sumitra19jha**
**What kind of change does this PR introduce?**
Closes #801 

**Did you add tests for your changes?**
No, a Separate issue is already created for it

**If relevant, did you update the documentation?**
Not required

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**

https://user-images.githubusercontent.com/35868598/119264471-00d6d300-bc01-11eb-8397-89560c44869a.mp4

### CODE REVIEW

**Comment by sumitra19jha** [comment]
Mentors, please review PR #829 first and after its merge, review this PR.

**Comment by sumitra19jha** [comment]
@palisadoes @Sagar2366 @DeltaHarbinger This PR is also ready for the review.

File: lang/zh-CN.json
Comment (TheHazeEffect) [inline lang/zh-CN.json:1]: Hi @sumitra19jha 
Why was this file deleted

**Comment by palisadoes** [comment]
@sumitra19jha merge this into the GSoC branch.

**Comment by sumitra19jha** [comment]
This is the Language Selector update for Master Branch.
Separate PR will come for GSoC Branch

**Comment by CyberWake** [comment]
@sumitra19jha all changes LGTM but please look into the test as this PR is based to master branch and there are few contributors still working to fix some bugs. Is this feature going to be implemented again for the GSoC branch codebase in another PR or will this we shipped as it is?

**Comment by sumitra19jha** [comment]
@CyberWake This PR is specifically targeted at the master branch. The new one is coming for the GSoC branch. Once this PR is merged, contributors will not use hardcoded string, this development quality will be enhanced.

**Comment by sumitra19jha** [comment]
@palisadoes @CyberWake @rutvik11062000 Please review

**Comment by palisadoes** [comment]
@CyberWake @rutvik11062000 @utkarshshendge Please review. We need 2 reviewers on the master branch to merge.

**Review by palisadoes — DISMISSED** [review]

**Comment by utkarshshendge** [comment]
@sumitra19jha #829 needs to be merged before this right?

**Comment by sumitra19jha** [comment]
@utkarshshendge Yes I would recommend that, but merging this PR first also won't effect the app working. But please go through string refactoring first.

**Comment by utkarshshendge** [comment]
@sumitra19jha  please resolve the conflicts.

**Comment by sumitra19jha** [comment]
@CyberWake @utkarshshendge @rutvik11062000 @palisadoes Conflict Resolved

**Review by palisadoes — APPROVED** [review]

**Comment by palisadoes** [comment]
@CyberWake @utkarshshendge @rutvik11062000 this needs one more review before it can be merged.

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**
